Library/Formula/makensis.rb

```diff
@@ -18,9 +23,17 @@ class Makensis < Formula
   # Use the right compiler by forcibly altering the scons config to set these
   patch :DATA
 
+  if build.devel?
```

Comment:
Please put these in a `stable do` and `devel do` block respectively, rather than making this a conditional. Thanks.